Macbook-setup.md

```diff
@@ -0,0 +1,163 @@
+# Mac - new system setup instructions
+
+These instructions describe software installations for MacOS as of 6/24/2021 and are based on an Intel-based Mac running Big Sur (macOS v11.4). Any problems with this process may result from a different architecture, OS version, or newer versions of the software being installed.
+
+_Note: MacOS uses Z shell (zsh) as the default shell. If you prefer bash, you can make that the default shell under System Preferences > Users and Groups (unlock the lock icon to make changes) > right click your user, select 'Advanced Options' and change 'Login shell' to /bin/bash. Then use ~/.bashrc instead of ~/.zshrc throughout this setup._
+
+## Download and install the following software:
+ - [Slack](https://slack.com/downloads/mac)
+ - [Zoom](https://zoom.us/download)
+ - [OneDrive](https://www.microsoft.com/en-us/microsoft-365/onedrive/download)*
+ - [Chrome](https://www.google.com/chrome/)*
+ - [LastPass for Chrome](https://lastpass.com/misc_download2.php)
+ - [VSCode](https://code.visualstudio.com/download)
+ - [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-mac.html)
+ - [Postgres.app](https://postgresapp.com/downloads.html) (get "Postgres.app with all currently supported versions" - we are currently running v10.x in production)
+ - [pgAdmin4](https://www.pgadmin.org/download/)
+
+\* _may already be installed by UW IT_
+
+---
+## Install Homebrew 
+
+_Note: this will also install the XCode command line tools._
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Also not a blocker, but you could add a link to https://docs.brew.sh/Installation as source of instructions, so it's a little more future proof.

Reply:
Done